### DISCUSSION TITLE
hosting the bank

### DISCUSSION THREAD

**Discussion by jester223** [DSJAS/DSJAS#35] [General Q&A] [answered]
so i want to run and host the bank from host computer in my vm. my question is what do i need to put in both hosts files?

**Comment by ejv2** [comment, marked as answer]
The /etc/hosts file in your VM needs to be changed to have the IP of your host computer. Your host computer's file doesn't need to be changed, but it won't do any harm if you do.

Note that the IP needs to be the local IP of the host computer (not the VM) for it to work. Look up "windows get local IP" for info on how to get it.

**Reply by ejv2** [reply]
More comprehensive overview here:

https://github.com/DSJAS/DSJAS/blob/master/docs/administration/A%20Guide%20to%20URL%20Spoofing.md
